### DISCUSSION TITLE
Is there a UMA demo deployed on cloud which I can use to learn and understand all the messages?

### DISCUSSION THREAD

**Discussion by arjunballa** [JanssenProject/jans#10496] [Q&A]
I recently learnt that Gluu is leading implementer of UMA protocol and PatientShare is powered by Gluu. Is there a cloud deployment which I can use to learn?

**Comment by nynymike** [comment]
Short answer is no. 

Janssen Server distributions have binaries for container and VM installation. We provide docker, helm, and terraform assets. But you have to use these to deploy on your own cloud. 

See the Installation section of https://docs.jans.io
